README.md

```diff
@@ -1 +1,194 @@
-# Leaderboard-Project
\ No newline at end of file
+# Leaderboard-Project
+<a name="readme-top"></a>
+<div align="center">
+  <img src="leaderboard-logo.png" alt="leaderboard logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Leaderboard Project</b></h3>
+
+</div>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+
+# 📖 "To-do List with webpack Project!" <a name="about-project"></a>
+
+
+Leaderboard is a simple project built to consume the leaderboard API. A user can add their name and their score which then will be displayed on the list ot the existing scores. This app is also built with module files based on various functionalities, the stacks used to build the project are HTML, CSS and JS.
+
+## 🛠 Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+<p>HTML</p>
+<p>CSS</p>
+<p>JS</p>
+
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+ <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/JS">JS</a></li>
+    <li><a href="#">WebpackDevServer</a></li>
+    <li><a href="#">HTMLWebpackPlugin</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+- **Navbar (menu-bar)**
+- **localStorage functionality**
+- **Add book functionality**
+- **Remove book functionality**
+- **Footer section**
+- **contact information**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
[OPTIONAL]
- [ ] Your project currently doesn't have a live demo and as such you either provide one or you remove this section. When removing this section also take note to remove the section from the headers at the  top of your readme file. this change is important so as to make your readme very professional and not have sections which are redundant :+1: